src/commands/moderations/WarningsCommand.ts

```diff
@@ -27,16 +29,16 @@ export default class WarningsCommand extends BaseCommand {
 			})
 			.setTimestamp()
 			.setFooter({ text: member.user.id });
+
 		for (const warning of warnings) {
-			if (string.length >= 1500) {
+			if (string.length + warning.warning.length >= 1500) {
 				embed.setDescription(string);
 				await interaction.followUp({ embeds: [embed] });
 				string = "";
 			}
-			string += `**ID: ${warning._id!} | Moderator: ${warning.moderator}**\n${
-				warning.warning
-			} - <t:${warning.date}:f>\n\n`;
+			string += `**ID: ${warning._id!} | Moderator: ${warning.moderator}**\n${warning.warning} - <t:${warning.date}:f>\n\n`;
```

Comment:
same prettier stuff here, could change settings if people want that, but really no need (+ it gives me better readability in other/some stuff)